### PR TITLE
Rearrange everything

### PR DESCRIPTION
**Pull request by electronwill**
Finally, this should put everything into the correct file structure and site navigation structure.

### CODE REVIEW

**Comment by asmeurer** [comment]
This breaks several links. We should maybe look at doing some redirects for the more common top-level pages. 

We should also probably figure out how to make the conda docs 404 page better.

**Comment by asmeurer** [comment]
And I just noticed that http://conda.pydata.org/docs/config.html is already broken.

File: docs/source/admin.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Admin
+=====
+
+.. raw:: html
+
+        <html><head><meta http-equiv="refresh" content="0; URL='http://conda.pydata.org/docs/install/central.html'" /></head><body></body></html>
```
Comment (asmeurer) [inline docs/source/admin.rst]: Will a relative url work?

**Comment by electronwill** [comment]
Good point. I just checked, and relative urls do work, so I'm now changing all redirect pages to use relative urls.

**Comment by asmeurer** [comment]
The redirects all seem to work. The organization looks good, and nothing seems to be accidentally deleted (I didn't check that very carefully, though). 

My only concern here is the cheatsheet. There's no way that I can edit it in the future if some change needs to be made, other than attempting to edit the pdf directly. Is there any way we can include a source file here?

**Comment by electronwill** [comment]
@asmeurer @kerrywatson1

I checked pretty carefully for accidental deletions, so hopefully there aren't any.

I just talked with Kerry and Amenity about the cheat sheet. The pdf is generated from an Adobe Illustrator .ai file, about 2 MB, currently in Google Drive, and Amenity is the one who took all the text and information and formatted it to look good and fit on a single printed page. Amenity thinks Adobe Illustrator is available to all employees, and says it's quite straightforward to edit a few characters in a command or something like that and then export it to a pdf. Making more significant changes would probably require Amenity's help to make it look right.

We could move the .ai file from google docs to github, although I don't know if it makes any difference.

Another possibility would be to copy the information from the cheat sheet and make it into an .rst which would then produce html on the site, and we could link from the html version to the pdf version for printing, but I'm not sure whether having an rst/html version of the .ai content is worth the extra complexity and effort of keeping them synced by hand whenever we make any changes.

**Comment by electronwill** [comment]
@asmeurer @kerrywatson1 

OK, the cheat sheet .ai is added as requested. Please look it over, let me know if anything needs to change, and go ahead and merge it if it's all good.

**Comment by kerrywatson1** [comment]
@asmeurer @electronwill it's great, quite an improvement, can't wait to see it live. Yes, please merge. :)

**Comment by asmeurer** [comment]
OK, let's merge it. The docs have been having some build issues, so this may not go live until they are fixed.

**Comment by asmeurer** [comment]
OK, I fixed the build issues. The changes are live.

**Comment by electronwill** [comment]
Great-- thanks!

**Pull request merged**